src/main/riddl/Organization/organization.riddl

```diff
@@ -30,66 +49,326 @@ context OrganizationContext is {
                 }
                 on command EditOrganizationInfo {
                     then set Draft.info to @EstablishOrganization.info
```

Comment:
Shouldn't this use `@EditOrganizationInfo.info` instead of `@EstablishOrganization.info`?

Reply:
You are correct. I am replacing the above with a more descriptive form of command handling. i.e.

on command EditOrganizationInfo {
  then "set draft info fields to reflect values in EditOrganizationInfo"
  and "yield event OrganizationInfoEdited"
}